doc/rbd/rbd-windows.rst

```diff
@@ -142,11 +142,35 @@ initializes a partition::
 Limitations
 -----------
 
+CSV support
+~~~~~~~~~~~
+
 At the moment, the Microsoft Failover Cluster can't use WNBD disks as
 Cluster Shared Volumes (CSVs) underlying storage. The main reason is that
 ``WNBD`` and ``rbd-wnbd`` don't support the *SCSI Persistent Reservations*
 feature yet.
 
+Hyper-V disk addressing
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
```

Comment:
nit: do we need this as a warning? It is already in the limitation section.

Though I don't have a strong opinion about this.

Reply:
I was thinking about putting some special emphasis due to the security implications.

Reply:
I would word it differently but having it highlighted is good idea.